features/longtasks.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
-name: Long Tasks API
-description: TODO
+name: Long tasks
+description: The `PerformanceLongTaskTiming` interface provides information about tasks that occupy the UI thread for 50 milliseconds or more.
```

For symmetry with the next few features:

```suggestion
description: The `longtask` performance event type and the `PerformanceLongTaskTiming` API provides information about tasks that occupy the UI thread for 50 milliseconds or more, a metric for input latency, delayed animations, and other sources of "jank."```